bin/app.rb

```diff
@@ -52,6 +79,7 @@ def create_book
     author = gets.chomp
     book = Book.new(title, author)
     @books << book
+    save_books
```

Comment:
- [ ] Kindly fix this error. I am unable to add new books to the JSON file.

  ![juso](https://user-images.githubusercontent.com/81468675/228820267-75868e14-11db-46b3-a08c-7db4582a7131.jpg)

Reply:
Hi @julie-ify 👋, I appreciate your time in reviewing our pull request. The `save_books` method was re-created, successfully removing the bug. Thank you 🙏.